src/content/en/updates/2019/08/native-file-system.md

```diff
@@ -0,0 +1,457 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The new Native File System API enables developers to build powerful web apps that interact with files on the users local device, like IDEs, photo and video editors, text editors, and more. After a user grants a web app access, this API allows web apps to read or save changes directly to files and folders on the users device.
+
+{# wf_published_on: 2019-08-20 #}
+{# wf_updated_on: 2019-08-20 #}
+{# wf_featured_image: /web/updates/images/generic/file.png #}
+{# wf_tags: capabilities,file,filesystem,native-file-system #}
+{# wf_featured_snippet: The new Native File System API enables developers to build powerful web apps that interact with files on the users local device, like IDEs, photo and video editors, text editors, and more. After a user grants a web app access, this API allows web apps to read or save changes directly to files and folders on the users device. #}
+{# wf_blink_components: Blink>Storage>FileSystem #}
+
+{# When updating this post, don't forget to update /updates/capabilities.md #}
+
+# The Native File System API: Simplifying access to local files {: .page-title }
+
+{% include "web/_shared/contributors/petelepage.html" %}
+
+<div class="clearfix"></div>
+
+<aside class="note">
+  The Native File System API (formerly known as the Writeable Files API), is
+  available behind a flag in Chrome 77 and later, and should begin an origin
+  trial in Chrome 78 (stable in October). It is part of our capabilities
+  project, and this post will be updated as the implementation progresses.
+</aside>
+
+## What is the Native File System API? {: #what-is-it }
+
+The new Native File System API enables developers to build powerful web apps
+that interact with files on the users local device, like IDEs, photo and video
+editors, text editors, and more. After a user grants a web app access, this
+API allows web apps to read or save changes directly to files and folders
+on the users device.
+
+<aside class="caution">
+  We've put a lot of thought into the design and implementation of the Native
+  File System API to ensure that people can easily manage their files. See the
+  <a href="#security-considerations">Security and permissions</a>
+  section of this post.
+</aside>
+
+## Current status {: #status }
+
+| Step                                       | Status                       |
+| ------------------------------------------ | ---------------------------- |
+| 1. Create explainer                        | [Complete][explainer]        |
+| 2. Create initial draft of specification   | [In progress][spec]          |
+| 3. Gather feedback & iterate on design     | [In progress][spec]          |
```

Comment:
We tended to **bold** the *in-progress* steps.

Reply:
I intentionally did not bold.